strawberryfields/parameters.py

```diff
@@ -90,17 +90,20 @@
 ~~~~~~~~~~~~
 
 """
-
 import numbers
 
+import mock
```

Comment:
This forces us to make `mock` a requirement, doesn't it?

Reply:
```suggestion
from unittest import mock
```